c2corg_ui/templates/helpers/view.html

```diff
@@ -412,7 +412,9 @@ <h3 class="heading show-phone outings" ng-click="mainCtrl.animateHeaderIcon($eve
   % for wp_category in ['waypoints', 'waypoint_children']:
     % if document.get('associations').get(wp_category):
       % for wp in document.get('associations').get(wp_category):
-        % if wp.get('geometry'):
+        % if wp.get('geometry') and\
+            not (document['doctype'] == 'routes' and wp['waypoint_type'] == 'virtual'):
+            ## do not display virtual waypoints in route map #1321
```

Comment:
Not sure it is need to add a reference to #1321 => actually it's not that obvious it's a github issue and the issue only says the same thing than the comment above.